main.py

```diff
@@ -1,10 +1,42 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as f:
+        json_data = json.load(f)
+    for player in json_data:
+        guild = json_data[player]["guild"]
+        if guild is not None:
+            player_guild = Guild.objects.get_or_create(
+                name=guild["name"],
+                description=guild["description"]
+            )
+        else:
+            player_guild = [None]
+        race = json_data[player]["race"]
+        player_race = Race.objects.get_or_create(
+            name=race["name"],
+            description=race["description"]
+        )
+        skills = race["skills"]
+        for skill in skills:
+            if skill["name"] not in Skill.objects.all().values_list("name"):
```

Comment:
you don't need this condition if you are using get_or_create. It will create if skill doesn't exists, and get if skill exists